lib/client/signalflow/signalflow_client.js

```diff
@@ -74,6 +74,52 @@ function SignalflowClient(apiToken, options) {
     };
   }
 
+  function liveTailRequest(opts) {
```

Comment:
Since this is part of the "public API" of the SignalflowClient, please add documentation for this method. In particular, describe what is expected to be in `opts`.

Reply:
done